src/repositories/recommendationsRepository.js

```diff
@@ -1,100 +0,0 @@
-import connection from '../database/database';
-
-export async function insertRecommendation({ name, youtubeLink }) {
-  await connection.query(
-    `
-    INSERT INTO recommendations
```

Comment:
recommendations tem como a coluna score NOT NULL

Essa query apresentará problemas